bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -725,9 +725,20 @@ private String getStatusMessage(PullRequestCheckIssueVisitor visitor,
         progressBody.append(makeCollapsible("Using Skara CLI tools", reviewUsingSkaraHelp()));
         progressBody.append(makeCollapsible("Using diff file", reviewUsingDiffsHelp()));
 
+        progressBody.append("\n\n### Webrev\n");
+        progressBody.append(getWebrevCommentLink());
         return progressBody.toString();
     }
 
+    private String getWebrevCommentLink() {
+        var webrevComment = comments.stream()
+                .filter(comment -> comment.author().username().equals("mlbridge[bot]"))
```

Comment:
We can't hard code the bot name here, it needs to be a configuration parameter for pr bot. See configuration of mlbridge for an example of how we configure user names to filter comments.

Reply:
Will fix it.